index.js

```diff
@@ -139,7 +139,7 @@ class NgxScriptsCli {
         const s = v.replace(/'/g, "\\'").replace(/\\"/g, '"');
         return `'${s}'`;
       });
-      env = `export default ${env};\n`;
+      env = `export const env = <{ [s: string]: string; }>${env};\n`;
     }
```

Comment:
in case this overexplicit ugliness makes you scratch your head...
It's a shutup for noImplicitAny for when any of the values are null.

Reply:
Why use a cast instead of an explicit type, ie: `export const env: { [s: string]: string; } = ${env};`?

Reply:
Yeah agreed - I try to avoid type assertions whenever possible too.

Initially was gonna do this instead, but then I thought it might give me a "type undefined is not assignable to string" or whatever that way, but TBH I didn't try it, and typescript still surprises me sometimes :P
I'll give it a try today and let you know.

Reply:
so I tried it and it does give that error, but only when strictNullChecks is enabled - which I think we don't have enabled by default.
But that's be easily avoided by adding a "| null" to the value type anyway.
I'll change it now :) good catch!